entries/g/gleam.io.json

```
@@ -0,0 +1,16 @@
+{
+  "Gleam": {
+    "domain": "gleam.io",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Authy"
```

```suggestion
      "totp"
```

They only offer TOTP.